cmd/skipper/main.go

```diff
@@ -30,9 +30,9 @@ package main
 
 import (
 	"flag"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Logrus itself claims "The core API is unlikely to change much but please version control your Logrus to make sure you aren't fetching latest master on every build." I don't think, we are doing this... Have you considered to do that?